### PR TITLE
fix(core): darkened circle icon to match dark theme

### PR DESCRIPTION
**Pull request by ashley-o0o**
Adjusted the disabled switch in dark mode so that the colors match the other dark mode switches. Also added extra switches to include option with check.
Screenshot 2023-06-07 at 10 30 48 AM

Closes https://github.com/patternfly/patternfly/issues/5622

### CODE REVIEW

**Comment by ashley-o0o** [comment]
What: Closes [#5622 ](https://github.com/patternfly/patternfly/issues/5622)

File: src/patternfly/components/Switch/examples/Switch.md [hunk 1]
```diff
@@ -100,6 +100,26 @@ cssPrefix: pf-v5-c-switch
   {{#> switch-label switch-label--id="switch-disabled-2-on" switch-label--modifier="pf-m-on" switch-label--attribute='aria-hidden="true"'}}Message when on{{/switch-label}}
   {{#> switch-label switch-label--id="switch-disabled-2-off" switch-label--modifier="pf-m-off" switch-label--attribute='aria-hidden="true"'}}Message when off{{/switch-label}}
 {{/switch}}
+
+
+
+<br/>
+<br/>
+{{#> switch switch--attribute='for="switch-with-icon-disabled-1"'}}
+  {{#> switch-input switch-input--id="switch-with-icon-1" switch-input--attribute='disabled checked'}}{{/switch-input}}
```
Comment (mcoker) [inline src/patternfly/components/Switch/examples/Switch.md:109]: The value here needs to match the value in the `for` attribute above.

```suggestion
  {{#> switch-input switch-input--id="switch-with-icon-disabled-1" switch-input--attribute='disabled checked'}}{{/switch-input}}
```

File: src/patternfly/components/Switch/examples/Switch.md [hunk 2]
```diff
@@ -100,6 +100,26 @@ cssPrefix: pf-v5-c-switch
...
+  {{#> switch-toggle}}
+    {{#> switch-toggle-icon}}{{/switch-toggle-icon}}
+  {{/switch-toggle}}
+{{/switch}}
+<br/>
+<br/>
+{{#> switch switch--attribute='for="switch-with-icon-disabled-2"'}}
+  {{#> switch-input switch-input--id="switch-with-icon-2" switch-input--attribute='disabled'}}
```
Comment (mcoker) [inline src/patternfly/components/Switch/examples/Switch.md:117]: Same here - this needs to match the `for` attribute above

```suggestion
  {{#> switch-input switch-input--id="switch-with-icon-disabled-2"" switch-input--attribute='disabled'}}
```

**Review by mcoker — CHANGES_REQUESTED** [review]
Good job on this! This matches the check and the circle, but like we discussed offline, it's probably best if we retain the disabled color of the circle and update the check to match.

Because we're looking to get this in ASAP, the way I would probably go about this would be to:
* Declare a new variable at the end of this list for the check icon that we will use to manipulate the color of the check icon when the switch is in the disabled state, and give it the same value as the disabled circle color  https://github.com/patternfly/patternfly/blob/2f153a8479371d82b6e547b0e66ff322333bad02/src/patternfly/components/Switch/switch.scss#L9-L13
  * new variable - `--#{$switch}__input--disabled__toggle-icon--Color: var(--#{$pf-global}--palette--black-150);`
* In this disabled CSS block, in between the switch toggle selector and `cursor` declaration, add a new declaration for the check icon color to be defined by the disabled icon variable we defined in the previous step (Just a nit - you probably need an empty line after the new variable and the `cursor` declaration) https://github.com/patternfly/patternfly/blob/2f153a8479371d82b6e547b0e66ff322333bad02/src/patternfly/components/Switch/switch.scss#L123-L124 
  * new declaration - `--#{$switch}__toggle-icon--Color: var(--#{$switch}__input--disabled__toggle-icon--Color);`
* In the dark theme stylesheet, add a new declaration for the disabled check icon color that matches the disabled circle color https://github.com/patternfly/patternfly/blob/2f153a8479371d82b6e547b0e66ff322333bad02/src/patternfly/components/Switch/themes/dark/switch.scss#L4-L9
  * new declaration - `--#{$switch}__input--disabled__toggle-icon--Color: var(--#{$pf-global}--disabled-color--100);`

**Review by mcoker — APPROVED** [review]
LGTM!

**Review by mceledonia — APPROVED** [review]

**Review by mattnolting — APPROVED** [review]
LPTM 👍

**Pull request merged**